kafka/README.md

```diff
@@ -9,53 +9,50 @@ Kafka is a distributed publish-subscribe messaging system that is designed to be
 
 It makes use of the following technologies:
 
-* [StatefulSets](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/) 
+* [StatefulSets](http://kubernetes.io/docs/concepts/abstractions/controllers/statefulsets/)
 * [Headless Services](http://kubernetes.io/docs/user-guide/services/#headless-services)
+* [Openshift Applier](https://github.com/redhat-cop/casl-ansible/tree/master/roles/openshift-applier)
+
+### OpenShift objects
+The openshift-applier will create the following OpenShift objects:
+* A Project named `kafka` (see [files/projects/projects.yml](files/projects/projects.yml))
+* Three ImageStreams `kafka`, `zookeeper` and `rhel` (see [files/builds/template.yml](files/builds/template.yml), [../zookeeper/files/builds/template.yml](../zookeeper/files/builds/template.yml) and [files/imagestreams/images.yml](files/imagestreams/images.yml))
+* Two BuildConfigs named `kafka` and `zookeeper` (see [files/builds/template.yml](files/builds/template.yml) and [../zookeeper/files/builds/template.yml](../zookeeper/files/builds/template.yml))
+* Four Services named `kafka`, `kafka-headless`, `zookeeper` and `zookeeper-headless` (see [files/deployments/template.yml](files/deployments/template.yml) and [../zookeeper/files/deployments/template.yml](../zookeeper/files/deployments/template.yml))
+* Two StatefulSets named `kafka` and `zookeeper` (see [files/deployments/template.yml](files/deployments/template.yml) and [../zookeeper/files/deployments/template.yml](../zookeeper/files/deployments/template.yml))
+
+*NOTE: This requires permission to create new projects and that the `kafka` project doesn't already exist*
```

Comment:
I think wrapping this in asterisks is preventing this from getting the proper formatting. It should look like the callouts here: https://github.com/redhat-cop/casl-ansible/blob/master/docs/PROVISIONING_OPENSTACK.md#openstack-specific-requirements